### PR TITLE
WSTEAMA-747 - Implement fetchpriority attribute for hero image on Homepage

### PR DESCRIPTION
**Pull request by louisearchibald**
Resolves JIRA [[747](https://jira.dev.bbc.co.uk/browse/WSTEAMA-747)]

Overall changes
======

Following on from implementing lazy loading on Homepages as part of work completed in JIRA [[734](https://jira.dev.bbc.co.uk/browse/WSTEAMA-734)], we now want to use Chrome's fetchpriority attribute to load the hero image on our homepages. (the first one)

Code changes
======

- _A bullet point list of key code changes that have been made._

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/components/Image/index.tsx [hunk 1]
```diff
@@ -164,6 +166,7 @@ const Image = ({
               width={width}
               height={height}
               css={styles.image}
+              fetchpriority={fetchpriority}
```
Comment (karinathomasbbc) [inline src/app/components/Image/index.tsx:169]: Don't forget to include the changes to the lint config file

File: src/app/pages/HomePage/index.test.tsx [hunk 2]
```diff
@@ -141,6 +141,20 @@ describe('Home Page', () => {
         }
       });
     });
+    it('Only the first image on a homepage has Fetch Priority set to HIGH ', () => {
```
Comment (karinathomasbbc) [inline src/app/pages/HomePage/index.test.tsx]: **nitpick**: whitespace
```suggestion

    it('Only the first image on a homepage has Fetch Priority set to HIGH ', () => {
```

File: src/app/components/Curation/HierarchicalGrid/index.tsx [hunk 3]
```diff
@@ -60,6 +60,9 @@ const HiearchicalGrid = ({
 
           const lazyLoadImages = !(isFirstPromo && isFirstCuration);
 
+          const fetchPriorityImages =
```
Comment (karinathomasbbc) [inline src/app/components/Curation/HierarchicalGrid/index.tsx]: Use the same name as the attribute?
```suggestion
          const fetchpriority =
```

**Review by karinathomasbbc — APPROVED** [review]
This is great - and love the fact that we had added tests too!

**Review by andrewscfc — APPROVED** [review]
LGTM, nice work!

**Pull request merged**